packages/webpack-cli/src/webpack-cli.ts

```diff
@@ -989,6 +989,17 @@ class WebpackCLI implements IWebpackCLI {
           "Path to the configuration to be extended (only works when using webpack-cli).",
         helpLevel: "minimum",
       },
+      {
+        name: "dot-env",
```

Comment:
How about `--use-dot-env`/`--read-dot-env`?

Reply:
Makes sense, I don't have strong opinion here, but I think it should be short and understable :smile:

Reply:
using --read-dot-env now